pkg/network/network.go

```diff
@@ -112,6 +112,7 @@ type DomainTemplateValues struct {
 	Name      string
 	Namespace string
 	Domain    string
+	SubDomain string
```

Comment:
Possibly not camelcase?
```suggestion
	Subdomain string
```